### ISSUE TITLE
Scroll Over Popup 

### ISSUE THREAD

**Issue by Whatsupcory** [fulcrumapp/fulcrum-styler#18]
Is there any way to add a scroll over popup option?

**Comment by MappingKat** [comment]
Yes that's called a tooltip. That option should be available in the the popup settings. I need to double check though.
